dockers/docker-fpm-frr/base_image_files/TSB

```diff
@@ -6,11 +6,21 @@ if [ "$EUID" -ne 0 ] ; then
   exit 1
 fi
 
-# If run on supervisor of chassis, trigger remote execution of TSB on all linecards
 if [ -f /etc/sonic/chassisdb.conf ]; then
-  rexec all -c "sudo TSB chassis"
+  CHASSIS_TSA_STATE_UPDATE="CHASSIS_APP_DB HMSET "BGP_DEVICE_GLOBAL\|STATE" tsa_enabled "false""
+  CONFIG_DB_TSA_STATE_UPDATE='{"BGP_DEVICE_GLOBAL":{"STATE":{"tsa_enabled": "false"}}}'
+  current_tsa_state="$(sonic-cfggen -d -v BGP_DEVICE_GLOBAL.STATE.tsa_enabled)"
+  if [[ $current_tsa_state  == false ]]; then
+    echo "Chassis is already in Normal mode"
+    logger -t TSB -p user.info "Chassis is already in Normal mode"
+  else
+    sonic-db-cli $CHASSIS_TSA_STATE_UPDATE
+    sonic-cfggen -a "$CONFIG_DB_TSA_STATE_UPDATE" -w
+    echo "Chassis Mode: Maintenance -> Normal"
+    logger -t TSB -p user.info "Chassis Mode: Maintenance -> Normal"
+  fi
   echo "Please execute \"rexec all -c 'sudo config save -y'\" to preserve System mode in Normal state after reboot\
```

Comment:
Same comment here

Reply:
@fountzou : Please address this comment.

Reply:
Thank you for comment @tjchadaga @abdosi . Fixed -- kindly see https://github.com/sonic-net/sonic-buildimage/pull/19527